tests/testthat/testdata/v4.0.1-tasks-fail-round-id-val-pattern.json

```diff
@@ -0,0 +1,190 @@
+{
+    "schema_version": "https://raw.githubusercontent.com/hubverse-org/schemas/main/v4.0.1/tasks-schema.json",
+    "rounds": [{
+            "round_id_from_variable": true,
+            "round_id": "round_id_var",
+            "model_tasks": [{
+                "task_ids": {
+                    "round_id_var": {
+                        "required": ["invalid-round-id-in-var-req"],
+                        "optional": [
+                            "24_25_covid",
+                            "invalid-round-id-in-var-opt1",
+                            "invalid-round-id-in-var-opt2"
+                            ]
+                    },
```

Comment:
The above contains 3 invalid round IDs spread across both required and optional properties